### ISSUE TITLE
Document connecting to kubernetes cluster

### ISSUE THREAD

**Issue by machow** [cal-itp/data-infra#668]
Once #633 is merged, add this to relevant section (kubernetes?) of docs:

```
source kubernetes/gke/config-cluster.sh
export KUBECONFIG=$HOME/.kube/data-infra-apps.yaml
gcloud container clusters get-credentials "$GKE_NAME" --region "$GKE_REGION"
```

(or something more relevant this is just a spell to me ;)

**Comment by charlie-costanzo** [comment]
@holly-g I think this can be closed as it was accomplished with #910

**Comment by holly-g** [comment]
@charlie-costanzo thanks-- should we store this screencast somewhere? https://loom.com/share/folder/20a1bdb3e5b04215a0d13e9e22439382

**Comment by charlie-costanzo** [comment]
@holly-g this new issue #938 adds the video to the docs!
